src/Client.php

```diff
@@ -74,10 +74,20 @@ public function __construct(Configuration $config, PipelineInterface $pipeline =
         $this->resolver = $resolver ?: new BasicResolver();
         $this->diagnostics = new Diagnostics($this->config, $this->resolver);
         $this->guzzle = $guzzle ?: new Guzzle(['base_uri' => static::ENDPOINT]);
+    }
 
-        // Register a shutdown function to check for fatal errors
-        // and flush any buffered errors
-        register_shutdown_function([$this, 'shutdownHandler']);
+    /**
+     * Destroy the client instance.
+     *
+     * @return void
+     */
+    public function __destruct()
```

Comment:
I'll revert this back to a shutdown function since php doesn't call destructors when there are fatal errors.